.github/workflows/postman-collection-runner.yml

```diff
@@ -0,0 +1,41 @@
+name: Postman Collection Api test 
+
+on:
+  schedule:
+      - cron: "15 0 * * *"
+  
+
+env:
+  TZ: 'Asia/Kolkata' 
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  api_tests:
+    name: Run Postman Collection on integ env
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.1.0
```

Comment:
Update the checkout version to 3.5.2

Reply:
Correction: Use `actions/checkout@v3`, we're only specifying major versions for official GitHub maintained actions.